content/docs/start/model-management/add-model.md

```diff
@@ -0,0 +1,102 @@
+---
+title: 'Get Started: Add a model'
+description: 'Start tracking model artifacts with DVC.'
+---
```

Comment:
Added a new page here to:
1. Consolidate info from https://dvc.org/doc/start/model-management/model-registry and https://dvc.org/doc/studio/model-registry
2. Make the first page less intimidating/quicker by focusing only on what's needed to add a model